signup.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="ko">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>판다마켓</title>
+		<link rel="stylesheet" href="style.css" />
+		<link rel="stylesheet" href="reset.css" />
+		<link rel="stylesheet" href="userForm.css" />
+		<link
+			rel="stylesheet"
+			as="style"
+			crossorigin
+			href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+	</head>
+	<body>
+		<main class="userForm_main">
+			<a href="/">
+				<img src="./image/logo.png" alt="판다마켓 로고" id="user_form_logo" />
+			</a>
+			<form action="#" class="user_form">
+				<label for="email">이메일</label>
+				<input type="email" id="email" name="email" placeholder="codeit@email.com" />
+				<label for="password">비밀번호</label>
+				<div class="input_box">
+					<input type="password" id="password" name="password" />
+					<img src="./image/btn_visibility_on.png" alt="비밀번호 보기" class="visible_on" />
+				</div>
+				<label for="password_check">비밀번호 확인</label>
+				<div class="input_box">
+					<input type="password" id="password_check" name="password_check" />
+					<img src="./image/btn_visibility_off.png" alt="비밀번호 안보기" class="visible_off" />
+				</div>
+				<button class="signup_button">회원가입</button>
```

Comment:
💊 제안
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
				<button class="signup_button" type="submit">회원가입</button>
```

+type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.

```html
// case1: form에 onsubmit 이벤트 명시
<form onsubmit="login">
  <input /> // input에서 enter 키 입력시 onsubmit 이벤트 실행
  <button type="submit">login</button> // 해당 버튼 클릭 시 onsubmit 이벤트 실행
</form>

// case2: submit 버튼에 onclick 이벤트 명시
<form>
  <input /> // input에서 enter 키 입력시 button의 onclick 이벤트 실행
  <button type="submit" onclick="login">login</button>
</form>
```

[mdn button](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#type)